tests/TestCase.php

```diff
@@ -1,9 +1,14 @@
 <?php
 
 use Rogue\Jobs\SendReportbackToPhoenix;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
 
 abstract class TestCase extends Illuminate\Foundation\Testing\TestCase
 {
+    use DatabaseMigrations;
+
+    static protected $isInitialTest = true;
```

Comment:
You can remove this for now! We added it when testing out that database transactions stuff, but it's not doing anything anymore.